openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTermTab/GlossaryTermTab.component.tsx

```diff
@@ -796,6 +821,12 @@ const GlossaryTermTab = ({
     return expandedRowKeys.length === expandableKeys.length;
   }, [expandedRowKeys, expandableKeys]);
 
+  useEffect(() => {
+    if (tableRef.current) {
+      setTableWidth(tableRef.current.offsetWidth);
+    }
+  }, []);
```

Comment:
Do we need this?

Reply:
Yes, we do since we are setting the column width based on table width which we are getting from Table Reference.  And changes in the ref value, don't re-render the document. So to make the change work have used `useEffect`, since it will trigger after a render and till then we will have tableRef value.

Reply:
You can use `tableRef.current` as deps

Reply:
Have tried that, but have not worked.